domain/src/main/kotlin/no/nav/su/se/bakover/domain/søknadsbehandling/Søknadsbehandling.kt

```diff
@@ -421,7 +451,31 @@ sealed class Søknadsbehandling : BehandlingMedOppgave, BehandlingMedAttestering
                     behandlingsinformasjon = behandlingsinformasjon,
                     grunnlagsdata = grunnlagsdata,
                     clock = clock,
-                )
+                ).let {
+                    if (vilkårsvurderinger.opplysningspliktVilkår() !is OpplysningspliktVilkår.Vurdert) {
+                        it.leggTil(
+                            /**
+                             * Vi legger til implisitt oppfylt for søknadsbehandling
+                             */
```

Comment:
Har vi noen tester som fanger opp at dette funker som det skal? F.eks at avslag pga manglende dokumentasjon ikke ender opp med innvilget for dette vilkåret?